bluebutton-data-model-rif/src/main/resources/db/migration/V12__Create_beneficiary_history_new_table.sql

```diff
@@ -0,0 +1,30 @@
+/*
+ * Creates the beneficiary ID history table required by the application. This
+ * schema is derived from the JPA metadata, via the 
+ * gov.hhs.cms.bluebutton.data.pipeline.rif.schema.HibernateSchemaPrinter 
+ * utility.
+ */
+
+${logic.tablespaces-escape} SET default_tablespace = pg_default;
```

Comment:
Is this the right tablespace? Let's make sure the temp table lands in the same tablespace as the real one.